smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/KafkaClientService.java

```diff
@@ -41,4 +43,18 @@
      */
     <K, V> KafkaProducer<K, V> getProducer(String channel);
 
+    /**
+     * Get the names of all the Kafka incoming channels managed by this connector.
+     *
+     * @return the names of the Kafka consumer incoming channels.
+     */
+    Set<String> getConsumerChannels();
+
+    /**
+     * Get the names of all the Kafka outgoing channels managed by this connector.
+     *
+     * @return the names of the Kafka producer outgoind channels.
```

Comment:
```suggestion
     * @return the names of the Kafka producer outgoing channels.
```

Reply:
Arg! I should stop doing three things at the same time, I know I'm not capable of doing multiple jobs in parallel.